datafusion/sqllogictest/test_files/insert_to_external.slt

```diff
@@ -40,8 +40,73 @@ STORED AS CSV
 WITH HEADER ROW
 LOCATION '../../testing/data/csv/aggregate_test_100.csv'
 
-# test_insert_into
 
+statement ok
+CREATE EXTERNAL TABLE dictionary_encoded_parquet
```

Comment:
I don't see any "dictionary" here -- why is it called `dictionary_encoded_parquet`?

Reply:
I assumed (incorrectly it seems) that `alltypes_dictionary.parquet` in the parquet-testing submodule was dictionary encoded. https://github.com/apache/parquet-testing/tree/e45cd23f784aab3d6bf0701f8f4e621469ed3be7/data

I will use your suggestion to generate dictionary encoded values instead.